src/main/java/software/amazon/cloudwatchlogs/emf/util/Validator.java

```diff
@@ -17,12 +17,17 @@
 package software.amazon.cloudwatchlogs.emf.util;
 
 import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
 import org.apache.commons.lang3.StringUtils;
 import software.amazon.cloudwatchlogs.emf.Constants;
 import software.amazon.cloudwatchlogs.emf.exception.*;
+import software.amazon.cloudwatchlogs.emf.model.StorageResolution;
 import software.amazon.cloudwatchlogs.emf.model.Unit;
 
 public class Validator {
+    public static Map<String, String> metricNameAndResolutionMap = new HashMap<>();
```

Comment:
Should be changed to static final constant or non-public and provide accessors

Reply:
I think this is not required in the validator class. Please refer the link : [Previous Review Comment](https://github.com/awslabs/aws-embedded-metrics-java/pull/129/files?short_path=b335630#r1054445785)

Reply:
Updated! Please take a look now.